.github/workflows/ci.yml

```diff
@@ -74,6 +80,20 @@ jobs:
           pytest -n auto --cov=autogpt --cov-report term-missing --cov-branch --cov-report xml --cov-report term
         env:
           CI: true
+          PROXY: ${{ vars.PROXY }}
+          AGENT_MODE: ${{ vars.AGENT_MODE }}
+          AGENT_TYPE: ${{ vars.AGENT_TYPE }}
 
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v3
+
+      - name: Stage new files and commit
+        run: |
+          git add .
```

Comment:
Specifically match files here

Reply:
Yeah, just pulling in the cassette dirs would be better